setup/initialize_database.sh

```diff
@@ -1,9 +1,36 @@
 #!/bin/sh
-echo "Setting up mysql database."
+echo "Setting up mysql database..."
 sudo mysql -uroot < acm.sql
-echo "Setting up submissions and problems data."
-mkdir ../auacm/app/data
-cp data.zip ../auacm/app/data/data.zip
-cd ../auacm/app/data
-unzip data.zip
-rm data.zip
+echo "Setting up test database..."
+echo "DROP DATABASE IF EXISTS acm_test; CREATE DATABASE acm_test; USE acm_test;\n" | mysql -uroot < acm.sql
+
+# Ask to purge the data directory if it exists
+if [ -e ../auacm/app/data ]
+then
+  echo "Do you want to reset the data directory? (y/n)"
+  read purge
+fi
+if [ "$purge"="y" ]
+then
+  echo "Purging and resetting data directory..."
+  rm -rf ../auacm/app/data
+  rmdir ../auacm/app/data
+fi
+
+# Create the data directory if it doesn't exist
+if [ ! -e ../auacm/app/data ]
+then
+  echo "Setting up submissions and problems data."
+  mkdir ../auacm/app/data
+  cp data.zip ../auacm/app/data.zip
+  cd ../auacm/app
+  unzip data.zip
```

Comment:
I know we didn't do this in the past, but it's kind of annoying to see all of the output for each file that is unzipped. Can we instead do `unzip data.zip > /dev/null`?